### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by shadow2560** [ITotalJustice/sphaira#2]
I've started to rewrite the french translation cause some translations were realy ununderstandable.

For Now it's incomplete, for some translations I need to see the context to make something more apropriate but this one should be better than the one made with automated tools.
[fr.json](https://github.com/user-attachments/files/18160306/fr.json)

**Comment by ITotalJustice** [comment]
> I've started to rewrite the french translation cause some translations were realy ununderstandable.
> 
> For Now it's incomplete, for some translations I need to see the context to make something more apropriate but this one should be better than the one made with automated tools. [fr.json](https://github.com/user-attachments/files/18160306/fr.json)

Hi, you can change the language in the options menu to English. maybe that would help with the context?

**Comment by shadow2560** [comment]
No, the problem is that I'm visualy impaired so reading on a screen is realy difficult, even impossible some times without a screen reader.

**Comment by ITotalJustice** [comment]
Ah okay. I'm happy to merge this as-is and leave this issue open until the translation is finished. Are you okay with sending a pull request?

**Comment by shadow2560** [comment]
Yes, I will make a pull request as soon as possible.

**Comment by Battosai94** [comment]
I adjusted some translations, and tried to keep typo.

[fr.json](https://github.com/user-attachments/files/18171386/fr.json)

Thank you for your work.

**Comment by ITotalJustice** [comment]
> I adjusted some translations, and tried to keep typo.
> 
> [fr.json](https://github.com/user-attachments/files/18171386/fr.json)
> 
> Thank you for your work.

hi, would you be okay to open a pull request with the changes?

**Comment by Battosai94** [comment]
> > I adjusted some translations, and tried to keep typo.
> > [fr.json](https://github.com/user-attachments/files/18171386/fr.json)
> > Thank you for your work.
> 
> hi, would you be okay to open a pull request with the changes?

I've to try Sphaira to see if it needs anothers changes, then I pull a request or if someone else want to do it he can.

**Comment by ITotalJustice** [comment]
closing as multiple pr's for fr has been submitted / merged
